foundation/org.eclipse.persistence.core/src/main/java/org/eclipse/persistence/internal/sessions/cdi/InjectionManagerImpl.java

```diff
@@ -47,7 +47,11 @@
     public InjectionManagerImpl(Object beanManagerInstance) throws NamingException {
         if (beanManagerInstance == null) {
             Context context = new InitialContext();
-            beanManagerInstance = context.lookup("java:comp/BeanManager");
+            try {
+				beanManagerInstance = context.lookup("java:comp/BeanManager");
+			} catch(Exception e) {
```

Comment:
why catching `Exception` where only `NamingException` is documented to be thrown?

Reply:
ok, i will change that